### DISCUSSION TITLE
Transfer Z2M to another "PC"

### DISCUSSION THREAD

**Discussion by DTMHibbert** [Koenkk/zigbee2mqtt#16210] [General]
Hi i currently have Z2M installed and running perfectly fine in a docker using UnRaid. Im looking to transfer this to a raspberry unit using the same sonoff USB dongle. Is this possible without having to start from scratch and repair every device? i currently have 60 devices which would be a nightmare to repair.

Thanks

**Comment by lux73** [comment]
afaik Copy 'data' Folder to the new PC and Change the new USB Path If necessary in your Configuration.yaml
